src/main/java/net/sf/jabref/gui/journals/ManageJournalAbbreviationsViewModel.java

```diff
@@ -0,0 +1,356 @@
+            }
+        }
+    }
+
+    /**
+     * Sets the name and the abbreviation of the {@code currentAbbreviation} property
+     * to the values of the {@code abbreviationsName} and {@code abbreviationsAbbreviation}
+     * properties.
+     *
+     * @throws EmptyFieldException if either the name or the abbreviation is empty
+     */
+    private void setCurrentAbbreviationNameAndAbbreviationIfValid() throws EmptyFieldException {
+        String name = abbreviationsName.get();
+        String abbreviation = abbreviationsAbbreviation.get();
+        if (name.trim().isEmpty()) {
+            throw new EmptyFieldException("Name can not be empty");
+        } else if (abbreviation.trim().isEmpty()) {
+            throw new EmptyFieldException("Abbrevaition can not be empty");
+        }
+        currentAbbreviation.get().setName(name);
+        currentAbbreviation.get().setAbbreviation(abbreviation);
+    }
+
+    /**
+     * Method to delete the abbreviation set in the currentAbbreviation property.
+     * The currentAbbreviationProperty will be set to the previous or next abbreviation
+     * in the abbreviations property if applicable. Else it will be set null.
+     */
+    public void deleteAbbreviation() {
+        if ((currentAbbreviation.get() != null) && !currentAbbreviation.get().isPseudoAbbreviation()) {
+            int index = abbreviations.indexOf(currentAbbreviation.get());
+            if (index > 1) {
+                currentAbbreviation.set(abbreviations.get(index - 1));
+            } else if ((index + 1) < abbreviationsCount.get()) {
+                currentAbbreviation.set(abbreviations.get(index + 1));
+            } else {
+                currentAbbreviation.set(null);
+            }
+            abbreviations.remove(index);
+        }
+    }
+
+    /**
+     * Calls the {@link AbbreviationsFileViewModel#writeOrCreate()} method for each file
+     * in the journalFiles property which will overwrite the existing files with
+     * the content of the abbreviations property of the AbbriviationsFile. Non
+     * existing files will be created.
+     */
+    public void saveJournalAbbreviationFiles() {
+        journalFiles.forEach(file -> {
+            try {
+                file.writeOrCreate();
+            } catch (IOException e) {
+                logger.debug(e.getLocalizedMessage());
+            }
+        });
+    }
+
+    /**
+     * This method stores all file paths of the files in the journalFiles property
+     * to the global JabRef preferences. Pseudo abbreviation files will not be stored.
+     */
+    public void saveExternalFilesList() {
+        List<String> extFiles = new ArrayList<>();
+        journalFiles.forEach(file -> {
+            if (!file.isBuiltInListProperty().get()) {
+                extFiles.add(file.getAbsolutePath());
+            }
+        });
+        Globals.prefs.putStringList(JabRefPreferences.EXTERNAL_JOURNAL_LISTS, extFiles);
```

Comment:
We are trying to reduce the occurrences of `Globals.prefs`. Thus I would like to see that the view model is independent on `Globals.prefs`. At least inject a JabRefPrefences class as a dependency and work on this class. Probably better it is to just work on an injected `JournalAbbreviationPreferences`. @oscargus does it make sense to introduce a field in `JournalAbbreviationPreferences` corresponding to `EXTERNAL_JOURNAL_LISTS`?